src/UIAutomation/UiaOperationAbstraction/UiaOperationAbstraction.h

```diff
@@ -2372,6 +2421,34 @@ namespace UiaOperationAbstraction
             GetCurrentDelegator()->If<OnTrue>(conditionBool, std::forward<OnTrue>(onTrue));
         }
 
+        template<class TryBody, class CatchBody>
+        inline void Try(TryBody&& tryBody, CatchBody&& catchBody) const
+        {
+            GetCurrentDelegator()->Try<TryBody, CatchBody>(std::forward<TryBody>(tryBody), std::forward<CatchBody>(catchBody));
+        }
+
+        template<class TryBody>
+        inline void Try(TryBody&& tryBody) const
+        {
+            GetCurrentDelegator()->Try<TryBody>(std::forward<TryBody>(tryBody));
+        }
+
+        UiaInt GetCurrentFailureCode() const 
+        {
+            auto currentFailureCode = GetCurrentDelegator()->GetCurrentFailureCode();
+            UiaInt resultFailureCode = S_OK;
+            if (!currentFailureCode.has_value())
+            {
+                resultFailureCode = wil::ResultFromCaughtException();
+            }
+            else
+            {
+                resultFailureCode = *currentFailureCode;

```

Comment:
**nit:** Just return inside these if/else blocks directly rather than assigning to a variable.